sync_diff_inspector/diff.go

```diff
@@ -157,6 +167,41 @@ func (df *Diff) AdjustTableConfig(cfg *Config) error {
 				continue
 			}
 
+			sourceTables := make([]TableInstance, 0, 1)
+			// find matched datbase name and table name in table router to fill source tables
+			for instanceID, allSchemas := range allTablesMap {
+				if instanceID == df.targetDB.InstanceID {
+					continue
+				}
+
+				for schema, allTables := range allSchemas {
+					for table := range allTables {
+						targetSchema, targetTable, err := df.tableRouter.Route(schema, table)
```

Comment:
can we iterate on **source** tables in outer `for` loop and try to find matched **target** table in inner `for` loop? Now, we have a 5-level `for` loop, a litter hard to read.

Reply:
done